ui/src/components/Footer.tsx

```diff
@@ -1,5 +1,5 @@
-import Image from 'next/legacy/image'
-import { ExternalLink, Links } from './ExternalLink'
+import { ExternalLink, Links } from '@/components/ExternalLink'
+import Image from 'next/image'
```

Comment:
why switching back to `next/image`

I had explicitly used `next/legacy/image` to fix some next warnings

https://github.com/anonklub/anonklub/pull/472/commits/48ab9424ba56d28ff8241b05f2c143f4a4c473cb

Can you confirm your change does not introduce new warnings?

Reply:
That was wrong change from me, thank you for noticing this. I reverted that change.